pipeline/inputs/node-exporter-metrics.md

```diff
@@ -37,6 +37,11 @@ This plugin is currently only supported on Linux based operating systems\
 | filesystem.ignore\_mount\_point\_regex  | Specify the regex for the mount points to prevent collection of/ignore. | `^/(dev|proc|run/credentials/.+|sys|var/lib/docker/.+|var/lib/containers/storage/.+)($|/)`    |
 | filesystem.ignore\_filesystem\_type\_regex  | Specify the regex for the filesystem types to prevent collection of/ignore. | `^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$`    |
 | diskstats.ignore\_device\_regex | Specify the regex for the diskstats to prevent collection of/ignore. | `^(ram|loop|fd|(h|s|v|xv)d[a-z]|nvme\\d+n\\d+p)\\d+$` |
+| systemd_service_restart_metrics | Determines if the collector will include service restart metrics | false |
+| systemd_unit_start_time_metrics | Determines if the collector will include unit start time metrics | false |
+| systemd_include_service_task_metrics | Determines if the collector will include service task metrics | false |
+| systemd_include_pattern | regex to determine which units are included in the metrics produced by the systemd collector | it's not applied unless explicitly set |
```

Comment:
I'm (apologies for the pun) not a huge fan of contractions like `it's` particularly when this is further confused by possessive forms. Let us try to ensure we use the full set of words, it is easy to accidentally misread or typo with a contraction.
```suggestion
| systemd_include_pattern | regex to determine which units are included in the metrics produced by the systemd collector | It is not applied unless explicitly set |
```